packages/neos-ui/src/Containers/LeftSideBar/NodeTreeSearchBar/NodeTreeSearchInput/style.css

```diff
@@ -1,34 +1,30 @@
 .searchInput {
     flex-grow: 1;
-}
 
-.searchInput {
-    /* magic number, sorry */
-    padding-left: 32px;
+    input {
+        padding-left: 40px;
+        border-radius: 0;
```

Comment:
you removed the "magic number, sorry" comment and replaced it with a new magic number :)
Can you elaborate?

Reply:
There's already a constant for 40px I think.

But by any means, don't be side-tracked by code styling for now, we can nitpick you to death here :stuck_out_tongue: 
Besides other things, I'd propose to focus on the visual aspect first, it's far more important.